product-scenarios/pom.xml

```diff
@@ -98,7 +98,7 @@
         <version.commons.logging>1.1.1</version.commons.logging>
         <identity.inbound.auth.saml.version>5.4.31</identity.inbound.auth.saml.version>
         <identity.user.ws.version>5.1.11</identity.user.ws.version>
-        <identity.inbound.auth.oauth.version>6.0.103</identity.inbound.auth.oauth.version>
+        <identity.inbound.auth.oauth.version>6.0.168</identity.inbound.auth.oauth.version>
```

Comment:
Why don't we update to the latest version?

Reply:
updated for 6.1.18